inlong-sort-standalone/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.inlong</groupId>
+        <artifactId>inlong</artifactId>
+        <version>0.12.0-incubating-SNAPSHOT</version>
+    </parent>
+    <packaging>pom</packaging>
+    <artifactId>inlong-sort-standalone</artifactId>
+    <name>Apache InLong - Sort - Standalone</name>
+
+    <modules>
+        <module>sort-standalone-source</module>
+        <module>sort-standalone-dist</module>
+    </modules>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <flume.version>1.9.0</flume.version>
+        <plugin.assembly.version>3.2.0</plugin.assembly.version>
+        <netty.version>3.8.0.Final</netty.version>
+        <codec.version>1.15</codec.version>
+        <servlet.version>2.5-20110124</servlet.version>
+        <inlong-common.version>1.3.4</inlong-common.version>
+        <pulsar.version>2.8.1</pulsar.version>
+        <testng.version>6.14.3</testng.version>
+    </properties>
+
+    <dependencies>
```

Comment:
add dependency according to actual usage.

Reply:
remove the unusable dependency.